packages/echo-base/src/errors/BaseError.test.ts

```diff
@@ -0,0 +1,68 @@
+import { BaseError, getAllProperties } from './BaseError';
+
+describe('BaseError', () => {
+    const message = 'this is a test';
```

Comment:
I think it's a good practice to provide somewhat realistic values when it comes to testing, so that it gives a bit more context.

In this case I think this would be a better one:
`const errorMessage = 'This is a custom error message for testing';`